frontend/micro-ui/web/micro-ui-internals/packages/modules/Estimate/src/configs/inboxConfig.js

```diff
@@ -140,7 +140,7 @@
                     links: [
                         {
                             text: "WORKS_CREATE_ESTIMATE",
-                            url: `/employee/project/search-project`,
+                            url: `/employee/estimate/create-estimate`,
```

Comment:
why create estimate it will be search project and select any project

Reply:
updated.